transport/http/handler.go

```diff
@@ -147,13 +149,11 @@ func handleOnewayRequest(
 	return nil
 }
 
-func updateSpanWithErr(span opentracing.Span, err error) error {
+func updateSpanWithErr(span opentracing.Span, err error) {
```

Comment:
So I figured just take it out, so you don't have to do a blank line to pass errcheck. It's just a helper function, and we may eventually want to use `-blank` with errcheck, right?